packages/react-components/react-menu/library/src/components/MenuItem/useMenuItemStyles.styles.ts

```diff
@@ -147,6 +147,9 @@ const useStyles = makeStyles({
   },
   disabled: {
     color: tokens.colorNeutralForegroundDisabled,
+    [`& .${menuItemClassNames.subText}`]: {
+      color: tokens.colorNeutralForegroundDisabled,
+    },
```

Comment:
https://github.com/microsoft/fluentui/blob/master/docs/react-v9/contributing/rfcs/react-components/styles-handbook.md#apply-classes-directly-to-elements